Intersect.Server.Framework/Plugins/Helpers/IMapHelper.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Intersect.Server.Framework.Items;
+using Intersect.Server.Framework.Maps;
+
+namespace Intersect.Server.Plugins.Helpers;
+
+public interface IMapHelper
+{
+    event Action<IItemSource, IItem> ItemAdded;
+    IMapInstance GetMapInstanceByDescriptorId(Guid mapId, Guid mapInstanceId);
+    void InvokeItemAdded(IItemSource source, IItem item);
```

Comment:
This shouldn't be on the interface because we only want the engine to be calling this